src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -512,9 +512,6 @@ const CheckoutLoadingManager: React.FC<{
   const [minimumLoadingPassed, setMinimumLoadingPassed] = useState(false)
   const [orderValidated, setOrderValidated] = useState(false)
 
-  const isExpressCheckoutLoaded = Order2CheckoutContext.useStoreState(
```

Comment:
I wish it would be easier to test Express Checkout locally. I just hope the express checkout loading won't be affected!